src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
+
+namespace NUnit.VisualStudio.TestAdapter
+{
+    public class AdapterSettings
+    {
+        private XmlNode _runConfiguration;
+        private XmlNode _testRunParameters;
+        private XmlNode _nunitAdapter;
+
+        #region Properties
+
+        public string MaxCpuCount
+        {
+            get { return GetInnerText(_runConfiguration, "MaxCpuCount"); }
+        }
+
+        public string ResultsDirectory
+        {
+            get { return GetInnerText(_runConfiguration, "ResultsDirectory"); }
+        }
```

Comment:
This is the setting used by VSTest to tell how many cores to use for running assemblies in parallel.  Do the adapter need this?   And, how do we separate out how many cores are available for NUnit parallel runs in itself ?